src/ol/featureloader.js

```diff
@@ -174,9 +174,10 @@ ol.featureloader.xhr = function(url, format) {
   return ol.featureloader.loadFeaturesXhr(url, format,
       /**
        * @param {Array.<ol.Feature>} features The loaded features.
+       * @param {ol.proj.Projection} dataProjectin Data projection.
```

Comment:
Should be `dataProjection`.  I've just confirmed that the `valid-jsdoc` rule catches this for a function declaration.  I'll dig a bit more to see why it isn't caught here.